modules/core/src/lib/deck.js

```diff
@@ -501,7 +510,7 @@ export default class Deck {
       onRender: this._onRenderFrame,
       onBeforeRender: props.onBeforeRender,
       onAfterRender: props.onAfterRender,
-      onError: props.onError
+      onError: onError && (error => onError(error, {operation: 'creating WebGL context'}))
```

Comment:
Does this duplicate the error messages created by luma?  I guess this makes sense for consistency, though this hits bundle size, and usage of these strings tends to be minimal in practice. 

An alternative is to create a new error with additional context in the cases where we feel we need to provide more information.

Reply:
> Does this duplicate the error messages created by luma? 

Yes, however not adding this additional `context` object will make the default `onError` handler more complex, which also hits bundle size. I can make the string more brief.

> An alternative is to create a new error with additional context in the cases where we feel we need to provide more information.

Doing that will lose the trace stack of the original error, which is valuable in debugging.